src/components/Header/Header.tsx

```diff
@@ -1,22 +1,13 @@
-import { Disclosure, Menu, Transition } from '@headlessui/react';
-import { HiBell, HiMenu, HiX } from 'react-icons/hi';
-import { Fragment } from 'react';
-import type { UserProps } from '../lib/types';
 import Link from 'next/link';
-import { supabase } from '../utils/supabaseClient';
-
-function classNames(...classes: Array<string>) {
-	return classes.filter(Boolean).join(' ');
-}
+import { HiBell, HiMenu, HiX } from 'react-icons/hi';
+import { Disclosure } from '@headlessui/react';
+import type { UserProps } from '../../lib/types';
+import { signOut } from '../../utils/auth';
+import { UserMenu } from './UserMenu';
 
 export function Header({ user }: UserProps) {
```

Comment:
Can you create a new `HeaderProps` interface for this here? `UserProps` has been renamed to `PageProps` in #161 and will have more page-specific props.

Reply:
@iShibi done, my friend, please check it out, I hope I understood your ask correctly.